pom.xml

```diff
@@ -234,6 +234,28 @@ This program is made available under the terms of the MIT License.
                 </executions>
             </plugin>
         </plugins>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.pitest</groupId>
+                    <artifactId>pitest-maven</artifactId>
+                    <version>1.1.11</version>
+                    <executions>
+                        <execution>
+                            <phase>verify</phase>
+                            <goals>
+                                <goal>mutationCoverage</goal>
+                            </goals>
+                            <configuration>
+                                <targetClasses>
+                                    <param>net.jadler*</param>
```

Comment:
This config is not necessary, since pitest assumes that your classes live in a package matching your projects group id (`<groupId>net.jadler</groupId>`).